posthog/api/prompt.py

```diff
@@ -34,20 +34,22 @@ def my_prompts(self, request: request.Request, **kwargs):
             )
             local_states.append(parsed_state)
 
+        team_id = 2  # we are testing prompts created by us, on PostHog Cloud
+
         my_prompts: Dict[str, Any] = {"sequences": [], "state": {}}
         states_to_update: List[PromptSequenceState] = []
         states_to_create: List[PromptSequenceState] = []
 
         person_id = (
-            PersonDistinctId.objects.filter(distinct_id=request.user.distinct_id, team_id=self.team_id)
+            PersonDistinctId.objects.filter(distinct_id=request.user.distinct_id, team_id=team_id)
```

Comment:
Note that usage of the hard-coded team ID makes this unusable on self-hosted instances. Do we have guards ensuring that this branch of the code can only run on Cloud?